user/anyrun/template.py

```diff
@@ -0,0 +1,26 @@
+pkgname = "anyrun"
+pkgver = "0.0.0"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo-auditable", "pkgconf"]
+makedepends = [
+    "gdk-pixbuf",
+    "glib-devel",
+    "gtk+3-devel",
+    "gtk-layer-shell-devel",
+    "pango-devel",
+    "rust-std",
+]
+pkgdesc = "Wayland native krunner-like runner"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/anyrun-org/anyrun"
+source = f"{url}/archive/refs/heads/master.tar.gz"
```

Comment:
Don't use "master", because the repository can change at any time. If you're using Git as the source, then use a specific commit.

Reply:
Ill be changing it then! Thanks!